api/src/Entity/Activity.php

```diff
@@ -60,10 +60,11 @@ class Activity extends AbstractContentNodeOwner implements BelongsToCampInterfac
     /**
      * The list of points in time when this activity's programme will be carried out.
      *
-     * @ORM\OneToMany(targetEntity="ScheduleEntry", mappedBy="activity", orphanRemoval=true)
+     * @ORM\OneToMany(targetEntity="ScheduleEntry", mappedBy="activity", orphanRemoval=true, cascade={"persist"})
      */
-    #[ApiProperty(writable: false, example: '["/schedule_entries/1a2b3c4d"]')]
-    #[Groups(['read'])]
+    #[Assert\Valid]
+    #[ApiProperty(writableLink: true, example: '["/schedule_entries/1a2b3c4d"]')]
```

Comment:
Das Example für OpenAPI solltest du noch anpassen, siehe [`Camp->$periods`](https://github.com/ecamp/ecamp3/blob/devel/api/src/Entity/Camp.php#L73). Nur die required Felder würden mir reichen, sodass man einen Startpunkt zum Copy Pasten hat in Swagger.

Reply:
done